build/cSpell.json

```diff
@@ -194,7 +194,8 @@
 	"xbind",
 	"xcprivacy",
 	"jlaban",
-	"sasakrsmanovic"
+	"sasakrsmanovic",
+	"maccatalyst"
```

Comment:
@jeromelaban I added maccatalyst in the ignore list related to https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=126873&view=logs&jobId=050fb64b-9b12-52cc-b750-ec5d4e778b08&j=050fb64b-9b12-52cc-b750-ec5d4e778b08&t=669790bd-240a-5aea-d705-86a7326c36e7